src/AssetManager.Web/Startup.cs

```diff
@@ -55,13 +72,13 @@ public void ConfigureProductionServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped(typeof(IAsyncRepository<>), typeof(EfRepository<>));
```

Comment:
This line is registering `EfRepository` to `IOC` container. This is required for controller to get repository via `constructor` injection. If you remove this line, your code will not run.